id-authentication-default.properties

```diff
@@ -665,4 +666,14 @@ mosip.ida.authentication.error.eventing.encrypt.partner.id=mpartner-default-temp
 mosip.ida.kafka.bootstrap.servers=kafka-0.kafka-headless.${kafka.profile}:${kafka.port},kafka-1.kafka-headless.${kafka.profile}:${kafka.port},kafka-2.kafka-headless.${kafka.profile}:${kafka.port}
 spring.kafka.admin.properties.allow.auto.create.topics=true
 logging.level.org.apache.kafka=DEBUG
+
+#Spring boot 3.x onwards we need to specify the below property to unmask values in actuator env url
+management.endpoint.env.show-values=ALWAYS
+spring.mvc.trailing-slash-match=false
+spring.batch.jdbc.initialize-schema=embedded
+spring.codec.max-in-memory-size=500KB
```

Comment:
why we are using this property

Reply:
management.endpoint.env.show-values=ALWAYS
this property is to show values in actutor so added in config if country wants to show that they can enable or disable .
this was added to fix api test rig issue as api test rig checking required fields for add identity api.

spring.mvc.trailing-slash-match=false
this property is added to fix websub issue where websub is added extra slash at end for callback api

spring.batch.jdbc.initialize-schema=embedded
https://stackoverflow.com/a/47947964/13425606
As of spring 2.5.0, we should use spring.batch.jdbc.initialize-schema